src/content/documentation/publish/submitting-an-add-on.md

```diff
@@ -113,6 +113,33 @@ Learn how to make your add-on listing appealing with these [tips](/documentation
 
 {% capture content %}
 
+## Version compatibility
```

Comment:
That's a fair point. I looked around but wasn't sure where else to put this information. I figured that modifying these values isn't part of the initial submission process, but it is part of updating an existing listing so maybe this was close enough.

Do you have a suggestion on what existing or new page should hold this information?

Reply:
@dotproto a lot depends on the reason why we're telling developers about this feature. In the extension workshop article on developing for Android we effectively recommend developers use the `browser_specific_settings` manifest key and explicitly set supported versions in there. Therefore, there should be no need to change this information during submission. This would point to including somewhere among the "update" articles - it doesn't seem to really fit into any of these pages - so a standalone page would be my suggestion.

Reply:
FYI: note sure if we want to mention that and where, but:

If you use `browser_specific_settings.gecko_android` compatibility for Android for that version is locked to whatever we extract from the manifest (filling in the gaps). Manifest becomes the source of truth for compatibility data when you do that, and the only way to change it is to submit a new version.

This is only true for `gecko_android`/Firefox for Android. We couldn't do that for `gecko`/Firefox for backwards-compatibility reasons.

Reply:
@diox, I'm a bit confused. You said that the Android version is locked to what's set in the manifest, but on AMO the tooltip for the "Compatibility" section on the version details page says (emphasis mine):

> Adjusting application information here will allow users to install your add-on <strong>even if the install manifest in the package indicates that the add-on is incompatible</strong> 

Does that mean the tooltip is also incorrect?

Reply:
> a lot depends on the reason why we're telling developers about this feature.
 
The main reason I wanted to call out this setting was because I thought it would be the most reliable way for developers to control Android version compatibility. Between your note about [Developing extensions for Firefox for Android](https://extensionworkshop.com/documentation/develop/developing-extensions-for-firefox-for-android/) encouraging `browser_specific_settings` and @diox's last comment, it seems like I may have been going down the wrong path.

Reply:
@dotproto no worries, just let me know when or if you need anything else

Reply:
We clarified that the tooltip is slightly misleading in a call - maybe worth filing an issue over addons-server to remove that bit from it

Reply:
I've opened https://github.com/mozilla/addons-server/issues/21366 to track the tooltip issue at @diox's request.